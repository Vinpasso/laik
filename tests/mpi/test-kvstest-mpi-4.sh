#!/bin/sh
LAIK_BACKEND=mpi ${MPIEXEC-mpiexec} -n 4 ../src/kvstest > test-kvstest-mpi-4.out
cmp test-kvstest-mpi-4.out "$(dirname -- "${0}")/test-kvstest-mpi-4.expected"
