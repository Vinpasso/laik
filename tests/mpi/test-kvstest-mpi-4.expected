Entries: 30
 [ 0] Key 'T0-d-0': 'from 0' (len 7)
 [ 1] Key 'T0-d-1': 'from 1' (len 7)
 [ 2] Key 'T0-d-2': 'from 2' (len 7)
 [ 3] Key 'T0-d-3': 'from 3' (len 7)
 [ 4] Key 'T0-v1': '1' (len 2)
 [ 5] Key 'T0-v2': '2' (len 2)
 [ 6] Key 'T1-d-0': 'from 0' (len 7)
 [ 7] Key 'T1-d-1': 'from 1' (len 7)
 [ 8] Key 'T1-d-2': 'from 2' (len 7)
 [ 9] Key 'T1-d-3': 'from 3' (len 7)
 [10] Key 'T1-v1': '1' (len 2)
 [11] Key 'T1-v2': '2' (len 2)
 [12] Key 'T2-d-0': 'from 0' (len 7)
 [13] Key 'T2-d-1': 'from 1' (len 7)
 [14] Key 'T2-d-2': 'from 2' (len 7)
 [15] Key 'T2-d-3': 'from 3' (len 7)
 [16] Key 'T2-v1': '1' (len 2)
 [17] Key 'T2-v2': '2' (len 2)
 [18] Key 'T3-d-0': 'from 0' (len 7)
 [19] Key 'T3-d-1': 'from 1' (len 7)
 [20] Key 'T3-d-2': 'from 2' (len 7)
 [21] Key 'T3-d-3': 'from 3' (len 7)
 [22] Key 'T3-v1': '1' (len 2)
 [23] Key 'T3-v2': '2' (len 2)
 [24] Key 'd-0': 'from 0' (len 7)
 [25] Key 'd-1': 'from 1' (len 7)
 [26] Key 'd-2': 'from 2' (len 7)
 [27] Key 'd-3': 'from 3' (len 7)
 [28] Key 'v1': '1' (len 2)
 [29] Key 'v2': '2' (len 2)
